src/__stories__/title-story.tsx

```diff
@@ -89,6 +89,18 @@ export const Title3Story: StoryComponent<Args> = ({title, right, linkText}) => (
     />
 );
 
+export const Title4Story: StoryComponent<Args> = ({title, right, linkText}) => (
+    <TitleComponent
+        as="h4"
```

Comment:
I think these `as` are wrong, bigger tittle (Title4) should have the h1

Reply:
Good catch. Fixed